### PR TITLE
Improve csv separator notes

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

For CSV seperaters, it is not recommended to use the default ",". It is recommended to use "|+|" or other uncommon character combinations

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/7504
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -371,7 +371,7 @@ Finally, all the exported data can be imported back to TiDB using [TiDB Lightnin
...
-| `--csv-separator`            | Separator of each value in CSV files                                                                                                                                                                                                                                                                                               | ','                                        |
+| `--csv-separator`            | Separator of each value in CSV files. It is not recommended to use the default ",". It is recommended to use "|+|" or other uncommon character combinations| ','                                                                                                                                                                                                                                                                                               | ','                                        |
```
Comment (Liuxiaozhen12) [inline dumpling-overview.md:374]: 这里预览有问题。可能是这个符号 "|+|" 的问题

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The CSV format can be configured in `tidb-lightning.toml` under the `[mydumper.c
 
 ```toml
 [mydumper.csv]
-# Separator between fields. Must not be empty.
+# Separator between fields. Must not be empty. Must be ASCII characters. It is not recommended to use the default ",". It is recommended to use "|+|" or other uncommon character combinations.
```
Comment (Liuxiaozhen12) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:32]:
```suggestion
# Separator between fields.  Must be ASCII characters. It is not recommended to use the default ",". It is recommended to use "|+|" or other uncommon character combinations.
```

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: tidb-lightning/migrate-from-csv-using-tidb-lightning.md [hunk 3]
```diff
@@ -191,7 +191,7 @@ The default setting is already tuned for CSV following RFC 4180.
 
 ```toml
 [mydumper.csv]
-separator = ','
+separator = ',' # It is not recommended to use the default ",". It is recommended to use "|+|" or other uncommon character combinations.
```
Comment (hfxsd) [inline tidb-lightning/migrate-from-csv-using-tidb-lightning.md:194]:
```suggestion
separator = ',' # It is not recommended to use the default ",". It is recommended to use "\|+\|" or other uncommon character combinations.
```

File: dumpling-overview.md [hunk 4]
```diff
@@ -371,7 +371,7 @@ Finally, all the exported data can be imported back to TiDB using [TiDB Lightnin
...
-| `--csv-separator`            | Separator of each value in CSV files                                                                                                                                                                                                                                                                                               | ','                                        |
+| `--csv-separator`            | Separator of each value in CSV files. It is not recommended to use the default ",". It is recommended to use "\|+\|" or other uncommon character combinations| ','                                                                                                                                                                                                                                                                                               | ','                                        |
```
Comment (hfxsd) [inline dumpling-overview.md:374]:
```suggestion
| `--csv-separator`            | Separator of each value in CSV files. It is not recommended to use the default ','. It is recommended to use '\|+\|' or other uncommon character combinations| ','                                                                                                                                                                                                                                                                                               | ','                                        |
```

File: dumpling-overview.md [hunk 5]
```diff
@@ -371,7 +371,7 @@ Finally, all the exported data can be imported back to TiDB using [TiDB Lightnin
...
-| `--csv-separator`            | Separator of each value in CSV files                                                                                                                                                                                                                                                                                               | ','                                        |
+| `--csv-separator`            | Separator of each value in CSV files. It is not recommended to use the default ‘,’. It is recommended to use ‘\|+\|’ or other uncommon character combinations| ','                                                                                                                                                                                                                                                                                               | ','                                        |
```
Comment (TomShawn) [inline dumpling-overview.md:374]:
```suggestion
| `--csv-separator`            |  Separator of each value in CSV files. It is not recommended to use the default ‘,’. It is recommended to use ‘\|+\|’ or other uncommon character combinations| ','                                                                                                                                                                                                                                                                                               | ','                                        |
```

**Review by Liuxiaozhen12 — APPROVED** [review]

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**